src/status_im/events.cljs

```diff
@@ -51,6 +51,7 @@
     status-im.wallet.choose-recipient.core
     [status-im.wallet.core :as wallet]
     status-im.wallet.custom-tokens.core
+    status-im2.contexts.emoji-picker.events
```

Comment:
not related to this, but we really need to clean up this events file. 
Why can't we have the core in src/status_im2/events.cljs and it requires in this file as `old-events` and then we require all new events into status-im2.events ?
Similiar to what we do for screens and old screens.

Or perhaps we already have this? 🤔 

cc @flexsurfer @ilmotta @briansztamfater

Reply:
I'm not sure I understand the question @J-Son89. But it looks like this namespace require should be in `status-im2.events`, not in `status-im.events`.

Reply:
Yes, I think it should be there too. 👍

Reply:
My bad, I placed in the wrong ns. It should be in `status-im2.events`. 

Also, I noticed we have several events from `status-im2` ns here. We should move those.